README.md

```diff
@@ -31,13 +34,16 @@ out:
   apikey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXX'
   list_id: 'XXXXXXXXXX'
   update_existing: false
+  double_optin: false
+  email_column: e mail
+  fname_column: first name
+  lname_column: last name
```

Comment:
@sakama it's the column name defined in MailChimp, it is valid. However, it's confusing to show as 'e mail'. @thangnc previous example in README used 'e-mail', can you keep it that way? Thanks!